gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/grouping/Grouping.java

```diff
@@ -81,6 +86,16 @@
    * See {@link LogicalGraph#groupBy(List, List, List, List, GroupingStrategy)}
    */
   public static final String LABEL_SYMBOL = ":label";
+
+  /**
+   * Used to verify if an aggregator is used for all vertices.
+   */
+  public static final String VERTEX_AGGREGATOR = ":vertexAggregator";
```

Comment:
If you add a vertex aggregator without specifying a certain label this aggregator is used for all vertices and to indicate this internally this constant is used.